src/components/Inputs.tsx

```diff
@@ -71,15 +104,16 @@ const Inputs: React.FC = () => {
     }
   };
 
-  const handleAddNewClaimClick = (event: React.FormEvent) => {
-    event.preventDefault();
-    setAddClaim(true);
+  const returnNewClaimsList = () => {
+    const claimsList = claims.reduce((acc: any, item: any) => {
```

Comment:
Why any here? I guess it is of type ClaimEntry?

Reply:
Changing it!